C_Controller/lowlevel/stm32f303/main.h

```diff
@@ -85,6 +85,11 @@ void Error_Handler(void);
 #define SWO_GPIO_Port GPIOB
 #define MOTOR_DIR_Pin GPIO_PIN_6
 #define MOTOR_DIR_GPIO_Port GPIOB
+#define SENSOR_SCL_Pin GPIO_PIN_8
+#define SENSOR_SCL_GPIO_Port GPIOB
+#define SENSOR_SDA_Pin GPIO_PIN_9
+#define SENSOR_SDA_GPIO_Port GPIOB
+
```

Comment:
Shouldn't this file be renamed to `lowlevel.h` ? That's unrelated to main (yeah, I know, this filename is inherited from HAL autogenerated files)

Reply:
OKI